docs/conf.py

```diff
@@ -112,6 +112,7 @@
     ("py:obj", "numpy._typing._array_like._ScalarType_co"),
     # https://github.com/sphinx-doc/sphinx/issues/10974
     ("py:class", "numpy.int64"),
+    ("py:class", "types.EllipsisType"),
```

Comment:
No idea what to do about this

Reply:
I’ll take a look.

By the way, the new `--exception-on-warning` flag is awesome to debug Sphinx stuff, finally this has become as easy as regular Python code.

Reply:
```suggestion
    # https://github.com/tox-dev/sphinx-autodoc-typehints/issues/498
    ("py:class", "types.EllipsisType"),
```